jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -160,6 +160,7 @@ Test splitting by timing does not work on Windows resource classes at this time.
 ## Next steps
 {: #next-steps }
 
+* How to [use the CircleCI CLI to split tests](/docs/use-the-circleci-cli-to-split-tests)
```

Comment:
If you're going to use the new link format, can you format the entire Next Steps section to all be the same?